### DISCUSSION TITLE
[Suggestion] Retire yeoman in favor of npm create

### DISCUSSION THREAD

**Discussion by Septh** [microsoft/vscode-discussions#1135] [Extension Development QnA] [answered]
While [yeoman](https://github.com/yeoman/yeoman) has been of great help to scaffold new extensions, the tool has not been updated in 4 years and will probably never be. Still, it is the tool that's recommended in the [Get Started](https://code.visualstudio.com/api/get-started/your-first-extension) guide to creating extensions for VS Code.

Maybe it is time to say "good bye and thanks for all" to Yeoman and switch to the modern `npm create` experience?

![image](https://github.com/microsoft/vscode-discussions/assets/11949623/77c19bfa-aefa-4429-91c0-f88729911285)

**Comment by isidorn** [comment, marked as answer]
Thanks for your feedback. We will keep this in mind. We might also explore Copilot as a way to generate new extensions.

fyi @aeschli
